higher-order-functions/min-max-sorted.md

```diff
@@ -0,0 +1,342 @@
+# Passing Functions as Arguments: Min, Max, and Sort
+
+## Learning Goals
+
+* Demonstrate how to pass a function as an argument
+* Describe how to use the key parameter in the `sorted` function to sort a list of dictionaries by a specific key
+* Describe how to use the key parameter in the `min` function to find the minimum in a list of dictionaries by a specific key
+
+## Introduction
+
+Imagine we have a list of movie titles we'd like to sort alphabetically. Python provides a couple convenient functions for this task. The [`sort`](https://docs.python.org/3/howto/sorting.html#sorting-basics) method can be called on a list and sorts the list in place. The `sorted` function takes a list as a parameter and returns the sorted list.  
+
+But what if we have a list of dictionaries representing movies that we want to sort by title, or perhaps by rating. Or perhaps we want to find the most recent movie or oldest movie in a list.
```

Comment:
The "or perhaps" repeated close together feels a little weird, I'd consider replacing one of them with "maybe" or "possibly".